test/foundry/unit/concrete/factory/TestK1ValidatorFactory_Deployments.t.sol

```diff
@@ -50,6 +50,29 @@ contract TestK1ValidatorFactory_Deployments is NexusTest_Base {
         assertTrue(isContract(address(factory)), "Factory should be a contract");
     }
 
+    /// @notice Tests that the constructor can take a zero address for the registry.
+    function test_ConstructorInitializesWithRegistryAddressZero() public {
```

Comment:
we would also have 7484 tests somewhere with some mock registry.
we can set it to zero and see if everything is going fine. 
cases I mentioned on slack thread once.

Reply:
Not sure which tests you mean